_posts/platform/app/2000-01-01-custom-error-page.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Custom error page
+modified_at: 2018-03-22 00:00:00
+tags: app custom error page
+---
+
+Our router may return an error page in various cases, when an application is unreachable:
+
+- The amount of container for the application is 0.
+- TODO no front == no web containers?!
+- The application returns a 5XX error.
```

Comment:
That's wrong actually, 5XX errors should be those if the application

Reply:
It's when the application crashes of cut the connection abrubptely (our router returns 502 + error page)

Reply:
No I mean if an app return 500, it's forwarded, it's not our error page